components/Auth/Services/client.js

```diff
@@ -12,3 +12,48 @@ export async function signUpUser(userData) {
   });
   return response.json();
 }
+//* Bug: simulator was working and users were able to be created but didn't work on mobile expo app
+//* Fix: Instead of local host, plugged in computer's IP address - theory is Backend isn't deployed
```

Comment:
This might be unrelated to the bug you're experiencing, but I think it's worth pointing out that for virtual devices iOS uses a simulator and android uses an emulator. This is an important distinction. https://osmancelik.medium.com/what-is-the-difference-between-a-simulator-and-an-emulator-b7f788bbd345

Because of this, the iOS simulator can access the host computers `localhost`. When you use `localhost` on the Android emulator, it's accessing the Android virtual device's local host. To access the host computer's localhost you have to use the alias`10.0.0.2` instead.